assets/src/edit-story/elements/image/layer.js

```diff
@@ -26,9 +26,10 @@ function ImageLayerContent({
     resource,
     resource: { alt },
   },
+  ...rest
 }) {
   const src = getSmallestUrlForWidth(0, resource);
-  return <VisibleImage src={src} alt={alt} height="20" />;
+  return <VisibleImage src={src} alt={alt} height="20" {...rest} />;
```

Comment:
Oh, this will conflict with the layer panel PR though... But we can deal with that later

Reply:
Yep, this will conflict, it's probably better if @samwhale gets his PR merged first since he is reworking the whole part. But either way is doable.